com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,39 @@
+%YAML 1.1
+
+---
+
+app-id: "com.deflemask.DefleMask"
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: "/app/share/DefleMask/DefleMask"
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
+  - "--device=dri"
+
+
+add-extensions:
+    'org.freedesktop.Platform.GL32':
+        directory: "lib/GL"
+        version: '18.08'
+        subdirectories: true
+        no-autodownload: true
+        autodelete: false
+        merge-dirs: "vulkan/icd.d;glvnd/egl_vendor.d"
+        download-if: active-gl-driver
+        enable-if: active-gl-driver
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/gtk2/gtk2.json
+  - app.yaml
+  - metadata.yaml
```

Comment:
Any reason for splitting those modules into separate files?

Reply:
Not particularly, come to think of it. I saw it done the same way on some other project, but since `app.yaml` isn't costly to rebuild I guess I could merge them.

Reply:
Most of those  files are generated automatically, hence splitting them makes sense to avoid cluttering the git log.

Reply:
then again, it makes both modules clearer to read individually -- `app` is _their_ files, `metadata` is _my_ files (in a way)

Reply:
This is a very straightforward manifest. There is no point in doing that, it actually impairs readability.

Reply:
so should I merge `app` and `metadata`, or place `metadata` in the parent manifest?

Reply:
I'd prefer placing both in the parent manifest.

Reply:
I pushed a WIP with the local modules in the main manifest